src/main/java/com/tripply/booking/service/FlightService.java

```diff
@@ -4,8 +4,13 @@
 import com.tripply.booking.model.request.FlightRequest;
 import com.tripply.booking.model.response.FlightResponse;
 
+import java.util.List;
+
 public interface FlightService {
 
 	ResponseModel<FlightResponse> addflight(FlightRequest flightRequest);
+	ResponseModel<List<FlightResponse>> getAllflights();
```

Comment:
don't return responseModel from service. return simply FlightResponse or List<FlightResponse> wherever is required.

Reply:
We are using the ResponseModel to maintain a standardized response format across all services.